appcues/src/main/java/com/appcues/debugger/DebuggerViewModel.kt

```diff
@@ -238,3 +252,26 @@ internal class DebuggerViewModel(
         _uiState.value = Expanded(Debugger(null))
     }
 }
+
+private fun Activity.captureScreen(
+    config: AppcuesConfig,
+    contextResources: ContextResources,
+) {
+    val timestamp = Date()
+    val view = window.decorView.rootView
+    val displayName = view.screenCaptureDisplayName(timestamp)
+    val layout = view.asCaptureView()
+    layout?.let {
+        val capture = Capture(
+            appId = config.applicationId,
+            displayName = displayName,
+            screenshotImageUrl = null,
+            layout = layout,
+            metadata = contextResources.generateCaptureMetadata(),
+            timestamp = timestamp,
+        )
+
+        // TESTING!!
+        capture.prettyPrint()
```

Comment:
this is where we'll eventually trigger the confirmation and then api data flow in some fashion.

thinking possibly this stuff lifts up into `AppcuesDebuggerManager` in some form, rather than in the VM file here.